db/seeds.rb

```diff
@@ -1,7 +1,31 @@
-# This file should contain all the record creation needed to seed the database with its default values.
-# The data can then be loaded with the bin/rails db:seed command (or created alongside the database with db:setup).
-#
-# Examples:
-#
-#   movies = Movie.create([{ name: 'Star Wars' }, { name: 'Lord of the Rings' }])
-#   Character.create(name: 'Luke', movie: movies.first)
+puts "Emptying database..."
+
+User.destroy_all
```

Comment:
can you add =>>>  if Rails.env.development? after both of these please @jonathansimpson101 , just means that they are not deleted from heroku when running the seed.Thanks